### DISCUSSION TITLE
do record classes improve microstream performance?

### DISCUSSION THREAD

**Discussion by guykatz** [microstream-one/microstream#572] [Q&A] [answered]
Hi;
I have seen in a microstream related presentation a reference for Record classes as something that might imporve microstream performance.
is this really the case?
thanks

**Comment by fh-ms** [comment, marked as answer]
Records are basically just syntax sugar to write short concise code for immutable data classes. There should be no difference at runtime in the JVM regarding performance. MicroStream itself sees Records the same way as normal classes.
